bgpd/bgp_vty.c

```diff
@@ -17495,7 +17495,21 @@ static struct cmd_node bgp_srv6_node = {
 
 static void community_list_vty(void);
 
-static void bgp_ac_neighbor(vector comps, struct cmd_token *token)
+static void bgp_ac_peergroup(vector comps, struct cmd_token *token)
+{
+	struct bgp *bgp;
+	struct peer_group *group;
+	struct listnode *lnbgp, *lnpeer;
+
+	for (ALL_LIST_ELEMENTS_RO(bm->bgp, lnbgp, bgp)) {
+		for (ALL_LIST_ELEMENTS_RO(bgp->group, lnpeer, group))
+			if (token->type == VARIABLE_TKN)
```

Comment:
`token->type` isn't going to change while executing the loops here, please move it up to save some CPU :) - only call `bgp_ac_peergroup` for `VARIABLE_TKN`

Reply:
Right :) Fixed, thanks, dude!